index.html

```diff
@@ -2286,10 +2286,13 @@ <h3>Presentations</h3>
           <dt><var id="defn-verifiableCredential">verifiableCredential</var></dt>
           <dd>
 The <code>verifiableCredential</code> <a>property</a> MAY be present. The value
-MUST be an array of one or more <a href="#verifiable-credential-graphs">
-verifiable credential graphs</a> in a cryptographically <a>verifiable</a> format.
-See Section <a href="#verifiable-credential-graphs"></a> for further details on
-this topic.
+MUST be one or more <a>verifiable credential</a> and/or
+<a href="#enveloped-verifiable-credentials">enveloped verifiable credential</a>
+objects. For the avoidance of doubt, the values MUST NOT be non-object values such as
+numbers, strings, or URLs. These types of objects are called
```

Comment:
Having the sentence "These types of objects" follow the avoidance of doubt sentence was a little confusing, so we could avoid this with a parenthetical:

```suggestion
objects (for the avoidance of doubt, the values MUST NOT be non-object values such as
numbers, strings, or URLs). These types of objects are called
```